docker-api/src/main/java/com/yahoo/vespa/hosted/dockerapi/Docker.java

```diff
@@ -12,6 +12,11 @@
  * and to avoid OSGi exporting those classes.
  */
 public interface Docker {
+    /**
+     * Must be called before any other method. May be called more than once.
+     */
+    void start();
```

Comment:
If called more than once... always in the same thread then?

Reply:
Yes.